programmers/두-정수사이의-합.js

```diff
@@ -0,0 +1,9 @@
+function solution(a, b) {
+    let result = 0;
+    const min = Math.min(a,b);
+    const max = Math.max(a,b);
+    
+    for (let i = min; i <= max; i++) result += i;
+    
+    return result;
+};
```

Comment:
```suggestion
function solution(a, b) {
  return (a + b) * (Math.abs(a - b) + 1) / 2 ;
}
```
이렇게도 풀 수 있답니다.  이는 등차수열의 합 공식을 이용한 건데요, 승범님 풀이가 틀렸다는 것이 아니라 문제의 규칙과 공식을 찾는다면 더 간결한 코드를 짤 수 있다는 걸 보여주고 싶었습니다 :)